tests/plugins/selectall/selectall.js

```diff
@@ -45,6 +45,34 @@
 				acceptableResults,
 				bender.tools.html.prepareInnerHtmlForComparison( bender.tools.selection.getWithHtml( editor ), htmlMatchingOpts )
 			);
+		},
+
+		'test selectall in source view': function() {
```

Comment:
The test look s quite complex, I would simplify it:
- The eventsRecorder is not needed, this test should check (according to its description) if `selectAll` works in source mode, no need to check for additional events.
- Editor creation can be moved to `bender.editors = { ... }`, like:
```
    editorSource: {
			name: 'test_source_mode',
			startupData: '<p>foo</p><p>bar</p>',
			config: {
				startupMode: 'source'
			}
		}
```
- ANd then test itself could be simplified to something like:
```
	var editor = this.editors.editorSource;

	editor.execCommand( 'selectAll' );

	assert.areSame( 'source', editor.mode, 'editor.mode' );
	if ( CKEDITOR.env.ie && CKEDITOR.env.version <= 8 ) {
		assert.areSame( document.selection.createRange().text.length, 20 );
	} else {
		assert.areSame( CKEDITOR.document.getActive().$.selectionStart, 0 );
		assert.areSame( CKEDITOR.document.getActive().$.selectionEnd, 20 );
	}
```